src/components/Actions/Actions.vue

```diff
@@ -373,6 +419,26 @@ import DotsHorizontal from 'vue-material-design-icons/DotsHorizontal.vue'
 
 const focusableSelector = '.focusable'
 
+const getActions = (ctx) => {
```

Comment:
What's the reason you put this function outside the component? It's only called once, and I think it should work just fine in the render function as well, no?

Reply:
I would keep it in the render function, to be honest. Looks cleaner to me.

Reply:
It is not true @raimund-schluessler 
Any content inside render will be called many times.

Any small changes in state, will run render again and again.
Because of that, is a problem call functions inside template functions.

Moving some parts of code to outside render is a performance improvement.

The Vue template compile does it in some parts of the template automatically.

Reply:
I am preparing a PR with these improvements.
To be true, `Action` is already refactored in my local branch, but I've intent to sent it in other moment.

Reply:
The function is anyway called in the render function every time something changes. I don't see a difference in putting it outside.

And the slot has to be re-evaluated on every render, otherwise changed slot content won't be noticed.

And it's not true that it's called on every minor change. E.g. opening the menu causes no re-render. Just put a console log inside and observe.